tests/src/python/test_qgsrelationeditwidget.py

```diff
@@ -190,7 +190,7 @@ def choose_linked_feature():
         btn.click()
         # magically the above code selects the feature here...
 
-        link_feature = next(self.vl_link.getFeatures(QgsFeatureRequest().setFilterExpression('"fk_book"={}'.format(f[0]))))
+        link_feature = list(next(self.vl_link.getFeatures(QgsFeatureRequest().setFilterExpression('"fk_book"={}'.format(f[0])))))
```

Comment:
What's the rationale here? This has changed the test from testing the attribute index 0 to testing that the feature is not None (which will never be the case)

Reply:
Sometimes I get stop iteration error on python 3 due to the use of next because it now returns and iterator, not a list. Since I've seen this on code and travis has thrown and stop iteration error, I thought it would be nice to fix it. If you like me to, I can change it back to what was written! =]

Reply:
Got it!